physics/ephemeris_body.hpp

```diff
@@ -63,6 +64,9 @@ using ::std::placeholders::_3;
 
 Time const max_time_between_checkpoints = 180 * Day;
 
+#define PRINCIPIA_USE_EXTENDED_GEOPOTENTIAL 0
```

Comment:
Define it to _DEBUG or something, so that something tries to at least compile the new path.